packages/runtime-common/realm.ts

```diff
@@ -603,7 +621,10 @@ export class Realm {
     }
 
     return new Response(JSON.stringify({ data }, null, 2), {
```

Comment:
this is an example of a response that should not be instantiate. There should be no new Response, rather the response that is used should be the one that is passed in.